23459 container save button enabled on edit

zulqarnainvd opened the pull request:
### Proposed Changes
* fixed container save button enable when visit container edit page again
* fixed after saving container button enable issue

### Checklist
- [x] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

zulqarnainvd committed:
dotCMS/core#23459 fix save button enable issue on edit

zulqarnainvd committed:
dotCMS/core#23459 update test case

alfredo-dotcms commented on core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.ts:
```diff
@@ -136,6 +136,8 @@ export class DotContainerPropertiesComponent implements OnInit {
         if (formValues.identifier) {
             this.store.editContainer(formValues);
+            this.store.updateOriginalFormState(formValues);
+            this.form.updateValueAndValidity();
```
test to check these functions are doing any changes!

zulqarnainvd replied:
Noted

zulqarnainvd replied:
test case added.

zulqarnainvd committed:
dotCMS/core#23459 add test case

zulqarnainvd committed:
Merge branch 'master' into 23459-container-save-button-enabled-on-edit

hmoreras commented on core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.spec.ts:
```diff
@@ -346,5 +346,15 @@ describe('DotContainerPropertiesComponent', () => {
+            fixture.detectChanges();
+            const saveBtn = de.query(By.css('[data-testId="saveBtn"]'));
+            saveBtn.triggerEventHandler('click');
+            tick(200);
```
is this necessary ?

zulqarnainvd replied:
Yes because we need to make sure after the save button is disabled.

zulqarnainvd commented:
@alfredo-dotcms @hmoreras Guys, I added a video that I fixed in this PR.

[screen-capture (1).webm](https://user-images.githubusercontent.com/113915849/208708629-c54f06f4-f754-4b5b-b25d-0232d743e939.webm)

zulqarnainvd committed:
Merge branch 'master' into 23459-container-save-button-enabled-on-edit
